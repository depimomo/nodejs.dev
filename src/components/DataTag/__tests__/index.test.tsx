import { render } from '@testing-library/react';
import React from 'react';

import DataTag from '..';

describe('Data Tag component', () => {
  it(`renders correctly when tag is 'E'`, () => {
    const { container } = render(<DataTag tag="E" />);
    expect(container).toMatchSnapshot();
  });

  it(`renders correctly when tag is 'C'`, () => {
    const { container } = render(<DataTag tag="C" />);
    expect(container).toMatchSnapshot();
  });

  it(`renders correctly when tag is 'M'`, () => {
    const { container } = render(<DataTag tag="M" />);
    expect(container).toMatchSnapshot();
  });
});
