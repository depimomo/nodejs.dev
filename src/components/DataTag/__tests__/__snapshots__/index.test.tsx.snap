// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Data Tag component renders correctly when tag is 'C' 1`] = `
<div>
  <span
    class="dataTag"
    data-tag="C"
  >
    C
  </span>
</div>
`;

exports[`Data Tag component renders correctly when tag is 'E' 1`] = `
<div>
  <span
    class="dataTag"
    data-tag="E"
  >
    E
  </span>
</div>
`;

exports[`Data Tag component renders correctly when tag is 'M' 1`] = `
<div>
  <span
    class="dataTag"
    data-tag="M"
  >
    M
  </span>
</div>
`;
